src/include/processor/operator/load.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "processor/operator/physical_operator.h"
+
+namespace kuzu {
+namespace processor {
+
+class Load final : public PhysicalOperator {
```

Comment:
I would rename this to `LoadExtension` to not be confused with `LoadFrom`, which is more explicit.